pycbc/inference/distributions.py

```diff
@@ -192,6 +193,42 @@ def _bounded_from_config(cls, cp, section, variable_args,
     return cls(**dist_args)
 
 
+def get_kde_from_file(params_file, params=None):
```

Comment:
I don't think it is thought to be used with anything else. @cdcapano, what do you think?